source/jormungandr/jormungandr/street_network/here.py

```diff
@@ -36,11 +36,26 @@
 import requests as requests
 from jormungandr import app
 from jormungandr.exceptions import TechnicalError
-from jormungandr.utils import get_pt_object_coord
+from jormungandr.utils import get_pt_object_coord, PeriodExtremity
 from jormungandr.street_network.street_network import AbstractStreetNetworkService, StreetNetworkPathKey
 from jormungandr.ptref import FeedPublisher
 from jormungandr.fallback_modes import FallbackModes as fm
 from six import text_type
+from enum import Enum
+
+MAX_MATRIX_POINTS = 100  # The limit is fixed by Here API
```

Comment:
If this is only relevant for Here, it should a member attribute to the Here class.

Reply:
@kadhikari proposed to add this little cute inside the default_value.py file. I added it